models/sipnet/R/model2netcdf.SIPNET.R

```diff
@@ -59,22 +59,27 @@ model2netcdf.SIPNET <- function(outdir, sitelat, sitelon, start_date, end_date,
     output[[5]] <- (sub.sipnet.output$rtot * 0.001) / timestep.s    # Total Respiration in kgC/m2/s
     output[[6]] <- (sub.sipnet.output$rAboveground * 0.001) / timestep.s +
       (sub.sipnet.output$rRoot * 0.001) / timestep.s                # Autotrophic Respiration in kgC/m2/s
-    output[[7]] <- (sub.sipnet.output$rSoil * 0.001) / timestep.s   # Heterotropic Respiration in kgC/m2/s
-    output[[8]] <- (sub.sipnet.output$nee * 0.001) / timestep.s     # NEE in kgC/m2/s
+    output[[7]] <- ((sub.sipnet.output$rSoil - sub.sipnet.output$rRoot) * 0.001) / timestep.s   # Heterotrophic Respiration in kgC/m2/s
+    output[[8]] <- (sub.sipnet.output$rSoil * 0.001) / timestep.s   # Soil Respiration in kgC/m2/s
+    output[[9]] <- (sub.sipnet.output$nee * 0.001) / timestep.s     # NEE in kgC/m2/s
     #output[[9]] <- rep(-999,sipnet.output.dims[1])             # CarbPools
-    output[[9]] <- (sub.sipnet.output$plantWoodC * 0.001)         # Above ground wood kgC/m2
-    output[[10]] <- (sub.sipnet.output$plantLeafC * 0.001)        # Leaf C kgC/m2
-    output[[11]] <- (sub.sipnet.output$plantWoodC * 0.001)+
+    output[[10]] <- (sub.sipnet.output$plantWoodC * 0.001)         # Above ground wood kgC/m2
+    output[[11]] <- (sub.sipnet.output$plantLeafC * 0.001)        # Leaf C kgC/m2
+    output[[12]] <- (sub.sipnet.output$plantWoodC * 0.001)+
       (sub.sipnet.output$plantLeafC * 0.001)+
       (sub.sipnet.output$coarseRootC * 0.001)+
       (sub.sipnet.output$fineRootC * 0.001)                       # Total living C kgC/m2
-    output[[12]] <- (sub.sipnet.output$soil * 0.001)+
+    output[[13]] <- (sub.sipnet.output$soil * 0.001)+
       (sub.sipnet.output$litter * 0.001)                          # Total soil C kgC/m2
-    output[[13]] <- (sub.sipnet.output$fluxestranspiration * 0.001) / timestep.s  #Transpiration kgW/m2/s
-    output[[14]] <- (sub.sipnet.output$soilWater * 10)            # Soil moisture kgW/m2
-    output[[15]] <- (sub.sipnet.output$soilWetnessFrac)         # Fractional soil wetness
-    output[[16]] <- (sub.sipnet.output$snow * 10)                 # SWE
-    output[[17]] <- sub.sipnet.output$litter * 0.001 ## litter kgC/m2
+    ## *** NOTE : npp in the sipnet output file is actually evapotranspiration, this is due to a bug in sipnet.c : ***
+    ## *** it says "npp" in the header (written by L774) but the values being written are trackers.evapotranspiration (L806) ***
+    ##  water density = 1000 kg m-3 , latent heat of vaporization = 2.501*10^6 J kg-1
+    output[[14]] <- (sub.sipnet.output$npp * 0.01 * 1000 * 2.501 * 10^6) / timestep.s  # Qle W/m2
```

Comment:
2. did you run roxygen explicitly? It should have updated the NAMESPACE automatically -- you don't need to do that by hand. The documentation for the function also wasn't generated, which is why I'm suspicious roxygen didn't run

3. don't pass that arbitrary airtemp value to get.lv, but rather set it as the default in the function definition so you don't have to pass anything unless you want to correct for airtemp.